docs/src/snap/reference/certificates.md

```diff
@@ -0,0 +1,35 @@
+### Cluster Certificates
+
+#### 1. **CertificateAuthorities (CA)**
+
+| **Common Name**         | **Usage**                            | **Path on Disk**                     | **Used For**                              |
+|------------------------------|--------------------------------------|--------------------------------------|-------------------------------------------|
+| `kubernetes-ca`               | General Kubernetes CA               | `/etc/kubernetes/pki/ca.crt`         | Signing all Kubernetes-related certificates |
+| `kubernetes-front-proxy-ca`   | CA for front-end proxy              | `/etc/kubernetes/pki/front-proxy-ca.crt` | Signing certificates for the front-proxy |
+| `client-ca`                   | CA for client certificates          | `/etc/kubernetes/pki/client-ca.crt` | Signing certificates for the client |
+
+
+#### 2. **Certificates**
+
+| **Common Name**                       | **Usage** | **Path on Disk**                                     | **Used For**                                                     | **Signed By**               |
+|--------------------------------------------|-----------|------------------------------------------------------|------------------------------------------------------------------|-----------------------------|
+| `kube-apiserver`                           | Server    | `/etc/kubernetes/pki/apiserver.crt`                  | Securing the API server endpoint                                 | `kubernetes-ca`             |
+| `kube-apiserver-kubelet-client`            | Client    | `/etc/kubernetes/pki/apiserver-kubelet-client.crt`   | API server communication with kubelets                           | `kubernetes-ca-client`      |
+| `kube-apiserver-etcd-client`               | Client    | `/etc/kubernetes/pki/apiserver-etcd-client.crt`      | API server communication with etcd                               | `kubernetes-ca-client`      |
+| `front-proxy-client`                       | Client    | `/etc/kubernetes/pki/front-proxy-client.crt`         | API server communication with the front-proxy                    | `kubernetes-front-proxy-ca` |
+| `kube-controller-manager`                  | Client    | `/etc/kubernetes/pki/controller-manager.crt`         | Communication between the controller manager and the API server  | `kubernetes-ca-client`      |
+| `kube-scheduler`                           | Client    | `/etc/kubernetes/pki/scheduler.crt`                  | Communication between the scheduler and the API server           | `kubernetes-ca-client`      |
+| `kube-proxy`                               | Client    | `/etc/kubernetes/pki/proxy.crt`                      | Communication between kube-proxy and the API server              | `kubernetes-ca-client`      |
```

Comment:
```suggestion
| `system:kube-proxy`                               | Client    | `/etc/kubernetes/pki/proxy.crt`                      | Communication between kube-proxy and the API server              | `kubernetes-ca-client`      |
```
https://github.com/canonical/k8s-snap/blob/main/src/k8s/pkg/k8sd/pki/control_plane.go#L43

Reply:
The actual CN that is written to the the certificates is `system:kube-proxy` 
https://github.com/canonical/k8s-snap/blob/main/src/k8s/pkg/k8sd/pki/control_plane.go#L268-L272

Arguably, I would as well need to update `kube-scheduler` etc. to be consistent.

Reply:
thanks, updated this and the other CNs (kube-scheduler etc).